tests/osgi-tests/src/test/java/org/wso2/carbon/osgi/startupresolver/PendingCapabilityRegistrationOSGiTest.java

```diff
@@ -59,30 +60,29 @@
      */
     @Configuration
     public Option[] createConfiguration() {
-
-        List<Option> optionList = new ArrayList<>();
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.mgt").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.http").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.jetty").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.custom2").groupId("org.wso2" +
-                ".carbon").versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.datasource.mgt").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.order.resolver").groupId("org.wso2.carbon")
-                .versionAsInProject());
-
-        optionList = OSGiTestConfigurationUtils.getConfiguration(optionList, null);
-        return optionList.toArray(new Option[optionList.size()]);
+        return new Option[] {
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.transport.mgt").groupId("org.wso2.carbon")
+                        .versionAsInProject()),
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.transport.http").groupId("org.wso2.carbon")
+                        .versionAsInProject()),
+                copyDropinsBundle(
+                        maven().artifactId("org.wso2.carbon.sample.transport.jetty").groupId("org.wso2.carbon")
+                                .versionAsInProject()),
+                copyDropinsBundle(
+                        maven().artifactId("org.wso2.carbon.sample.transport.custom2").groupId("org.wso2.carbon")
+                                .versionAsInProject()),
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.datasource.mgt").groupId("org.wso2.carbon")
+                        .versionAsInProject()),
+                copyDropinsBundle(maven().artifactId("org.wso2.carbon.sample.order.resolver").groupId("org.wso2.carbon")
+                        .versionAsInProject())
+        };
```

Comment:
All the osgi tests are following the same pattern.
[1] https://github.com/wso2/carbon-datasources/blob/master/tests/osgi-tests/src/test/java/org/wso2/carbon/datasource/osgi/DataSourceManagementServiceTest.java

[2] https://github.com/wso2/carbon-jndi/blob/master/tests/osgi-tests/src/test/java/org/wso2/carbon/jndi/osgi/JNDITest.java